test/stdlib/Span/SpanTests.swift

```diff
@@ -0,0 +1,627 @@
+  reason: "Requires Swift 6.2's standard library"
+))
+.code {
+  guard #available(SwiftStdlib 6.2, *) else { return }
+
+  let capacity = 4
+  let a = Array(0..<capacity)
+  a.withUnsafeBufferPointer {
+    let span = Span<Int>(_unsafeElements: $0)
+    expectEqual(span.count, capacity)
+    expectEqual(span._extracting(last: capacity)[0], 0)
+    expectEqual(span._extracting(last: capacity-1)[0], 1)
+    expectEqual(span._extracting(last: 1)[0], capacity-1)
+    expectEqual(span._extracting(droppingFirst: capacity).count, 0)
+    expectEqual(span._extracting(droppingFirst: 1)[0], 1)
+  }
+
+  do {
+    let b = UnsafeBufferPointer<Int>(start: nil, count: 0)
+    let span = Span(_unsafeElements: b)
+    expectEqual(span.count, b.count)
+    expectEqual(span._extracting(last: 1).count, b.count)
+    expectEqual(span._extracting(droppingFirst: 1).count, b.count)
+  }
+}
+
+suite.test("withUnsafeBufferPointer")
+.skip(.custom(
+  { if #available(SwiftStdlib 6.2, *) { false } else { true } },
+  reason: "Requires Swift 6.2's standard library"
+))
+.code {
+  guard #available(SwiftStdlib 6.2, *) else { return }
+
+  let capacity: UInt8 = 64
+  let a = Array(0..<capacity)
+  a.withUnsafeBufferPointer {
+    ub in
+    let span = Span(_unsafeElements: ub)
+    span.withUnsafeBufferPointer {
+      let i = Int.random(in: 0..<$0.count)
+      expectEqual($0[i], ub[i])
+    }
+
+    let emptyBuffer = UnsafeBufferPointer(rebasing: ub[0..<0])
+    expectEqual(emptyBuffer.baseAddress, ub.baseAddress)
+
+    let emptySpan = Span(_unsafeElements: emptyBuffer)
+    emptySpan.withUnsafeBufferPointer {
+      expectNil($0.baseAddress)
+    }
+  }
+}
+
+suite.test("withUnsafeBytes()")
+.skip(.custom(
+  { if #available(SwiftStdlib 6.2, *) { false } else { true } },
+  reason: "Requires Swift 6.2's standard library"
+))
+.code {
+  guard #available(SwiftStdlib 6.2, *) else { return }
+
+  let capacity: UInt8 = 64
+  let a = Array(0..<capacity)
+  a.withUnsafeBufferPointer {
+    ub in
+    let span = Span(_unsafeElements: ub)
+
+    span.withUnsafeBytes {
+      let i = Int.random(in: $0.indices)
+      expectEqual($0.load(fromByteOffset: i, as: UInt8.self), ub[i])
+    }
+
+    let emptyBuffer = UnsafeBufferPointer(rebasing: ub[0..<0])
+    expectEqual(emptyBuffer.baseAddress, ub.baseAddress)
+
+    let emptySpan = Span(_unsafeElements: emptyBuffer)
+    emptySpan.withUnsafeBytes {
+      expectNil($0.baseAddress)
+    }
+  }
+}
+
+suite.test("borrowing 1")
+.skip(.custom(
+  { if #available(SwiftStdlib 6.2, *) { false } else { true } },
+  reason: "Requires Swift 6.2's standard library"
+))
+.code {
+  guard #available(SwiftStdlib 6.2, *) else { return }
+
+  let capacity = 8
+  Array(0..<capacity).withUnsafeBufferPointer {
+    var span = Span(_unsafeElements: $0)
+    let prefix = span._extracting(0..<2)
+    span = span._extracting(1...)
+    expectEqual(span.count, capacity-1)
+    expectEqual(prefix.count, 2)
+  }
+}
+
+suite.test("borrowing 2")
```

Comment:
This looks like a repeat of the previous test.

Reply:
It is. Removed as well.